utils/python/CIME/system_test.py

```diff
@@ -149,12 +150,12 @@ def __init__(self, test_names,
                                                                self._baseline_gen_name)
 
             # Compute baseline_root
-            if baseline_root is None:
-                self._baseline_root = self._machobj.get_value("CCSM_BASELINE")
-            else:
-                self._baseline_root = baseline_root
+            self._baseline_root = baseline_root if baseline_root is not None \
```

Comment:
I think this is a reasonable way to split a ternary across two lines if we must.